js/crm.menubar.js

```diff
@@ -55,6 +55,13 @@
               // For empty links - keep the menu open and don't jump the page anchor
               return false;
             })
+            .on('click', 'a', function(e) {
+              if (! $(this).attr('href').match(/^#/)
```

Comment:
jQuery selectors are pretty powerful. I think you can actually skip this conditional and change the handler to

    .on('click', 'a:not([href^="#")', function(e) {

haven't actually tried it but that seems like it should work.

Reply:
Cool. FYI https://api.jquery.com/attribute-starts-with-selector/